docs/angular-testing-library/faq.mdx

```diff
@@ -0,0 +1,107 @@
+---
+id: faq
+title: FAQ
+---
+
+See also the [main FAQ](dom-testing-library/faq.mdx) for questions not specific
+to Angular testing.
+
+<details>
```

Comment:
This and the 3rd item are identical copies from React's FAQ.
I think these items should apply to all frameworks.
Should keep them on the Angular FAQ as well?
If positive, should we deduplicate them somehow? Maybe by moving them to the main FAQ or by creating a reusable doc snippet.